_XiGetDevicePresenceNotifyEvent is referenced to in liba11y-keyboard.so,
but library is not linked to libXi

--- gnome-settings-daemon-2.32.1/plugins/a11y-keyboard/Makefile.am.~1~	2015-03-18 01:26:26.581033126 +0300
+++ gnome-settings-daemon-2.32.1/plugins/a11y-keyboard/Makefile.am	2015-03-18 01:27:18.271395556 +0300
@@ -63,6 +63,7 @@
 	$(SETTINGS_PLUGIN_LIBS)		\
 	$(XF86MISC_LIBS)		\
 	$(LIBNOTIFY_LIBS)		\
+	$(XINPUT_LIBS)			\
 	$(NULL)
 
 plugin_in_files = 		\
